### PR TITLE
some typos in map.txt

### PR DESCRIPTION
**Pull request by Phil-Morley**
Fixed 5 typos in map.txt of 004_Shields_up/data

(and hopefully worked out how to use github :)

### CODE REVIEW

**Comment by Phil-Morley** [comment]
I changed the pirate frigate and spearhead to be medium warships and uploaded to my fork, seems that it has included these changes with this PR?

**Review by FriendlyNeighbour — COMMENTED** [review]
What were the fixes?

**Comment by Amazinite** [comment]
Instead of updating the existing files, you uploaded copies of the original files with your changes...

**Comment by Phil-Morley** [comment]
sorry that is what I thought I was supposed to do :) will try again :)

**Comment by Phil-Morley** [comment]
ok, fixed some typos and spelling errors in my fork and pull request. (deleted the uploaded files, which is also now showing here, just to really highlight that I am a noob on github :)

File: 004_Shields Up/data/map.txt [hunk 1]
```diff
@@ -9635,8 +9635,8 @@ planet "Epica Citadel"
...
-	spaceport `You land in one of the many small towns scattered across the planet. Even though it's a small town, the technologically advanced Alphas go all out with their constructions, the largest structure here being a large, five story high spire with beams sticking out of the side which are supporting landing pads.`
+	description `A beautifully calm planet owned by the Plenipotent Dominion. The main source of food for their planets, Exemption is rightfully named for the relaxing getaway from the typical Dominion planet due to its lack of factories. Instead, Exemption is the breadbasket of the Dominion, producing most of its food.`
+	spaceport `You land in one of the many small towns scattered across the planet. Even though its a small town, the technologically advanced Alphas go all out with their constructions, the largest structure here being a large, five story high spire with beams sticking out of the side which are supporting landing pads.`
```
Comment (Amazinite) [inline 004_Shields Up/data/map.txt]: This last one should remain "it's"

**Comment by Amazinite** [comment]
All the "refueling" -> "refuelling" changes are wrong. It's proper the way it was. 
Other than that, everything looks good.

File: 004_Shields Up/data/map.txt (hunk 1 above)
Comment (Phil-Morley) [reply 004_Shields Up/data/map.txt]: ok

**Comment by Phil-Morley** [comment]
American spelling? when I went back to change the file I couldn't remember the 5 changes I'd made so I dropped the text in word for the spell checker. Resisted putting the 'u's in :D

**Comment by FriendlyNeighbour** [comment]
I personally use the English spelling, so using a "u" is what I'd do

**Comment by Amazinite** [comment]
Yeah, I'm fine with extra u's.

**Comment by FriendlyNeighbour** [comment]
You cant spell Shields-Up without u :P

**Pull request merged**